### ISSUE TITLE
Broken in macOS 14.0 (Sonoma) public beta

### ISSUE THREAD

**Issue by jinhao-luo** [kofigumbs/multi#116]
Got `Expecting an OBJECT FIELD named "tabs"` in an empty setting window(see attachment) when opening a Multi app.
The app was working in macOS 13 (Ventura)

macOS: `14.0 Beta (23A5286i)`
<img width="1008" alt="Screenshot 2023-07-16 at 1 39 49 PM" src="https://github.com/kofigumbs/multi/assets/24256607/2fef900c-b843-4046-8e1c-4c9695293647">

**Comment by kofigumbs** [comment]
This doesn't seem related to the macOS version. The config in your picture is invalid since it doesn't have a `"tabs"` field. You need to have a valid JSON config in order to create an ap.

**Comment by jinhao-luo** [comment]
@kofigumbs That seems to be part of the problem. If I look into `<path-to-my>.app/Contents/Resources/config.json`, the config seems correct (and looks like this):
```
{
  "tabs": [
    {
      "title": "<my>",
      "url": "https://<app>.com"
    }
  ]
}
```

**Comment by kofigumbs** [comment]
Ahh gotcha, I misunderstood.

**Comment by kofigumbs** [comment]
Could you please try recreating your app in [v3.0.0](https://github.com/kofigumbs/multi/releases/tag/v3.0.0) and let me know if the issue persists?

- Download/install the new release
- Recreate the app, by opening it's Preferences and hitting save (or by running create-mac-app as described in the README)

**Comment by jinhao-luo** [comment]
Issue is resolved after recreating new apps with v3.0.0 Thanks @kofigumbs! ❤️
